### ISSUE TITLE
Reseek vs. Foldseek

### ISSUE THREAD

**Issue by Pooryamb** [steineggerlab/foldseek#381]
Hi Foldseek developers,

Sorry if it is not the best place to ask this question.
Recently, Reseek has been developed by Robert C. Edgar and claims to be more sensitive than Foldseek. Unlike Foldseek which transforms a 3D structure into a single 3Di vector, it transforms the 3D structure into multiple vectors. The rationale is that if the number of states in a single vector is so high, the substitution matrix derivation process which is based on very limited data, will probably miss out on the low-frequency state conversions. He claims that is why Foldseek's performance in a higher number of states was not better than when 20 states were used. However, when the structure is projected into multiple vectors, which he calls "matrix factorization", this problem does not happen. 

I was curious if you have had the chance to investigate this issue. 

From my point of view, this claim might be investigated by establishing a more comprehensive training dataset (taking advantage of AlphaFold structures), to see if the diminishing performance of a higher number of states is really related to the data scarcity. If making a new gold standard is not a trivial task, investigation of the optimal number of states with subsets of SCOPe can show if training data is the main reason why a higher number of states gives a lower performance.

If that claim is confirmed, I think the combination of Foldseek and Reseek might lead to a more sensitive structural alignment tool. I mean the structural state clustering approach of Foldseek along with transforming the structure into multiple vectors could take advantage of both approaches. 

Thanks for your feedback,

**Comment by martin-steinegger** [comment]
Several neural network approaches have demonstrated that using larger alphabets enables accurate structural reconstruction with minimal error, suggesting that structural information can indeed be encoded within a single alphabet. For example, see ESM3. Alternatively, using multiple alphabets can also achieve this, as seen with Foldtoken1-4. However, this approach increases storage demands and makes searching through large databases more resource-intensive.
